sync/index.markdown

```diff
@@ -23,28 +23,28 @@ Below are the various GitHub repositories that are part of the Data-Sync feature
 
 ### Data-Sync Server
 
-Our Netty-based [Java Server](https://github.com/aerogear/aerogear-sync-server) exposes WebSocket and XMPP/GCM endpoint for our different clients.
+* [Java Sync Server Engine](https://github.com/aerogear/aerogear-sync-server/tree/master/server/server-core#aerogear-server-differential-synchronization-server-core) the Server Sync Engine.
+* [Netty Sync Server](https://github.com/aerogear/aerogear-sync-server/tree/master/server/server-netty#netty-differential-synchronization-server-implementation) which exposes WebSocket and XMPP/GCM endpoint for our different clients.
+* [Node Sync Server](https://github.com/aerogear/aerogear-nodejs-sync-server#aerogear-nodejs-sync-server-) which exposes WebSocket for our different clients.
 
 ### Android
-
-An [XMPP-client library](https://github.com/aerogear/aerogear-android-sync) to receive sync updates over GCM.
+* [Java Client Sync Engine](https://github.com/aerogear/aerogear-sync-server/tree/master/client/client-core#aerogear-server-differential-synchronization-client-core) a Java Client Sync Engine implementation.
+* [Netty Sync Client](https://github.com/aerogear/aerogear-sync-server/tree/master/client/client-netty) a Netty based WebSocket Sync Client.
+* [XMPP Client](https://github.com/aerogear/aerogear-android-sync#xmpp-differential-synchronization-client-implementation) to receive sync updates over GCM.
+* [Demo application](https://github.com/danbev/android-diffsync-demo) which can be used against the above Java server.
```

Comment:
This is one of the links that I was referring to in my https://github.com/aerogear/aerogear.org/pull/498#issuecomment-75520410 above. I think the demo will be in the android cookbook but is not there yet. Should I just point to the the Android Cookbook?